### DISCUSSION TITLE
Upload to s3 - Failed to connect

### DISCUSSION THREAD

**Discussion by MoallaMed** [Budibase/budibase#6299] [Help]
![image](https://user-images.githubusercontent.com/102292139/173519034-5a8aa1ad-7913-4a9e-9033-f0b9d2d4b421.png)

I am trying to upload to AWS s3 in eu-west-1 on a self-hosted docker . 
Getting the following errors in my console:

budibase-client.js?v=1.0.145:243 Refused to connect to 'https://-----.s3.eu-west-1.amazonaws.com/--- Amz-SignedHeaders=host' because it violates the document's Content Security Policy.
Please advise

**Comment by melohagan** [comment]
Hi @MoallaMed 

It looks like you need to update. You're on version 1.0.145, but this has been fixed in the latest version!

**Reply by MoallaMed** [reply]
@melohagan  thank you for your reply
![image](https://user-images.githubusercontent.com/102292139/173580108-ac49eb15-ddc7-4079-a508-409d55e71b91.png)

I installed the latest version "1.0.199" on a minikube kubernetes cluster but a new error appears .
Can you please tell me what are the exact steps to do an upload in s3.

**Reply by melohagan** [reply]
Make sure your CORS configuration for your S3 bucket is correct: https://docs.aws.amazon.com/AmazonS3/latest/userguide/enabling-cors-examples.html
